public/js/editor.js

```diff
@@ -193,6 +207,15 @@
     if (window.document) {
         editorMain();
     } else {
-        window.addEventListener("DOMContentLoaded", editorMain);
+        window.addEventListener("load", editorMain);
     }
+    var target = document.querySelector('title');
+    var observer = new MutationObserver(function(mutations) {
+        mutations.forEach(function(mutation) {
+            if (document.title != 'Code Editor · EmulatorJS' && window.location.pathname == '/editor') {
```

Comment:
This can be done differently string comparison can be messy, though we should always use type comparison

Reply:
I changed it to this: document.title !== 'Code Editor · EmulatorJS' && window.location.pathname === '/editor'.

Reply:
But why are we comparing strings at all? Why are we not moving it to the init function with the rest of the init code?